.gitignore

```diff
@@ -1,3 +1,13 @@
+# Only vendor the net-op CRD for networkinterfaces.
```

Comment:
How about this instead so we don't have to update it as/if NetOP gains CRDs?

```
config/crd/external-crds/netoperator.vmware.com_*
!config/crd/external-crds/netoperator.vmware.com_networkinterfaces.yaml
```

Reply:
Good call!